public/robots.txt

```diff
@@ -1 +1,2 @@
-# See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
+User-agent: *
+Disallow: /support
```

Comment:
As I understand it, the problem with robots.txt is that if Google finds a link to /support somewhere, it will _index_ the page, but never visit it, so it won't know that the robots `<meta>` doesn't allow indexing. It will therefore keep it in index.

This is what leads to service domains being displayed without description:

<img width="728" alt="Screenshot 2020-02-18 at 14 48 34" src="https://user-images.githubusercontent.com/233676/74746730-c47a3180-525d-11ea-8845-7de312601c77.png">